PerfTools/JeProf/plugins/JeProfService.cc

```diff
@@ -123,10 +100,6 @@ using namespace edm::service;
 
 JeProfService::JeProfService(ParameterSet const &ps, ActivityRegistry &iRegistry)
     : mineventrecord_(1), prescale_(1), nrecord_(0), nevent_(0), nrun_(0), nlumi_(0), nfileopened_(0), nfileclosed_(0) {
-  if (!have_jemalloc_and_prof) {
-    edm::LogWarning("JeProfModule") << "JeProfModule requested but application is not"
-                                    << " currently being profiled with jemalloc profiling\n";
```

Comment:
This warning looks potentially useful. How about adding another function to `cms::jeprof` namespace, e.g. `isAvailable()`, that returns the value of `have_jemalloc_and_prof`?